examples/postman/postman_env_setup.py

```diff
@@ -0,0 +1,120 @@
+import json
+import os
+from pathlib import Path
+import sys
+
+# setup path variables
+pwd = os.getcwd()
+collection_path = Path(f'{pwd}/out/postman/collection.json')
+
+# pull generated collection from output file
+with open(collection_path, 'r') as content:
+    collection_json = json.loads(content.read())
+
+# setup preliminary environment variables
+env = json.loads("""
+[
+    {
+        "key": "baseUrl",
+        "value": "http://localhost:80",
+        "enabled": true
+    },
+    {
+        "key": "account",
+        "value": "dev",
+        "enabled": true
+    },
+    {
+        "key": "default-auth",
+        "value": "authn",
+        "enabled": true
+    },
+    {
+        "key": "role",
+        "value": "admin",
+        "enabled": true
+    },
+    {
+        "key": "api-key",
+        "value": "",
+        "enabled": true
+    },
+    {
+        "key": "token",
+        "value": "",
+        "enabled": true
+    },
+    {
+        "key": "formatted-token",
+        "value": "Token token=\\"{{token}}\\"",
+        "enabled": true
+    }
+]
+""")
+
+# update with command line args
+admin_api_key = sys.argv[1]
+admin_token = sys.argv[2].replace('\r','')[28:-1]
+env[4]['value'] = admin_api_key
+env[5]['value'] = admin_token
+collection_json["variable"] = env
+
+# insert pre-request authorization into collection json as
+# collection-wide pre-request event
+pre_event = json.loads("""
```

Comment:
When `json.loads` reads an array as the top level JSON object it will just return a Python `list` so this is really just a redundant way of declaring a `list` with a few `dicts` in it. You don't need to use the JSON library at all.